Clients/GUI/Pages/SignUp.razor

```diff
@@ -0,0 +1,59 @@
+﻿@page "/signup"
+@using GUIModels
+@using GUI.Scripts
+@inject NavigationManager NavigationManager
+
+    <div class="col-12">
+        <h3><b>Sign up</b></h3>
+        <hr />
+
+        <EditForm Model="@userInput" OnValidSubmit="@TrySignUp">
```

Comment:
Why do we have thos name? All validation named as ValidateSomething. It is understandable that we will try to do it

Reply:
If you mean `<DataAnnotationsValidator />`, it's .net stuff